recipes/jupyter_app_launcher/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "jupyter_app_launcher" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8d31694b6921e29f5f587f89764a0f1a7c61259cb01f14c62356def9cd31d787
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - jupyter-packaging >=0.12
+  run:
+    - python
+    - voila >=0.3.0,<0.4
```

The metadata upstream does not constrain it to <0.4
```suggestion
    - voila >=0.3.0
```

---

I'm playing safe here since we are going do a major breaking change in `Voila` 0.4